JSDemos/Demos/Toolbar/Multiline/description.md

```diff
@@ -1,5 +1,7 @@
-## Configure Toolbar Items
+This demo demonstrates the multiline mode in the Toolbar component. 
 
-## Specify Item Location
+To see step-by-step instructions on how to get started with the DevExtreme Toolbar component, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/). 
 
-## Customize Item Appearance
+In this demo, you can drag the Toolbar container's resize handle. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, you'll see how the Toolbar wraps onto multiple lines. 
+
+In single-line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), the Toolbar doesn't wrap. It can display an overflow menu with items that didn't fit into the container. Use the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property to control whether an item appears in the overflow menu.
```

Comment:
```suggestion
In single-line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), the Toolbar does not wrap. It can display an overflow menu with items that do not fit into the container. Use the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property to control whether an item appears in the overflow menu.
```